package.json

```diff
@@ -55,21 +70,32 @@
     "jest-dom": "^1.12.0",
     "lint-staged": "^7.2.0",
     "nock": "^10.0.0",
+    "openapi3-ts": "^0.12.0",
     "parcel": "^1.10.3",
     "prettier": "^1.13.5",
     "react-dom": "^16.4.2",
     "react-testing-library": "^5.0.0",
+    "request": "^2.88.0",
```

Comment:
Is there a reason you chose this over something like [isomorphic-unfetch](https://www.npmjs.com/package/isomorphic-unfetch) or [node-fetch](https://www.npmjs.com/package/node-fetch)?